VideoLocker/res/layout/fragment_discussion_thread_posts.xml

```diff
@@ -4,12 +4,13 @@
     android:orientation="vertical"
     android:layout_width="match_parent"
     android:layout_height="match_parent"
-    android:background="@color/white">
+    android:background="@color/edx_grayscale_neutral_x_light" >
 
     <LinearLayout
         android:layout_width="match_parent"
         android:layout_height="@dimen/discussion_post_refine_layout_height"
-        android:orientation="horizontal">
+        android:orientation="horizontal"
+        android:background="@color/edx_grayscale_neutral_white">
```

Comment:
Why set it's background to be the same as it's parent? Transparency is already the default.